lib/properties.js

```diff
@@ -29,9 +29,8 @@ class Properties extends EventEmitter {
       // Get Dynamic layers' source instances
       .concat(this.layers.map((layer) => layer.source).filter((source) => !!source))
 
-      // Add the active View's sources. Reversed for precedence consistency with Layers:
-      // Entry 0 is most precedent, LAST is least precedent.
-      .concat(this.active.sources.reverse());
+      // Add the active View's sources.
+      .concat(this.active.sources);
```

Comment:
Do we want to keep the API unchanged for `/v1/status` and have this go out as a bug fix? If so, we can write this as
```javascript
return []
  .concat(this.active.sources)
  .concat(this.layers.map((layer) => layer.source).filter((source) => !!source))
  .reverse();
```